src/status_im/constants.cljs

```diff
@@ -503,11 +503,18 @@
 (def ^:const send-type-stickers-buy 4)
 (def ^:const send-type-bridge 5)
 (def ^:const send-type-erc-721-transfer 6)
+(def ^:const send-type-erc-1155-transfer 7)
 
 (def ^:const bridge-name-transfer "Transfer")
 (def ^:const bridge-name-erc-721-transfer "ERC721Transfer")
+(def ^:const bridge-name-erc-1155-transfer "ERC1155Transfer")
 (def ^:const bridge-name-hop "Hop")
 
+(def ^:const contract-type-uknown 0)
```

Comment:
should we specify what type of contract? 🤔

Reply:
That value is for unknown contract: https://github.com/status-im/status-go/blob/develop/services/wallet/common/const.go#L34

Reply:
I more meant, this is in a very high level file. For that reason contract could mean a plethora of things.
Perhaps smart_contract_... is more appropriate

Reply:
I think a prefix of `wallet` also works, just to stay similar to the go side which uses add its under the wallet namespace.

Reply:
Yep probably better tbh